### PR TITLE
Send version information to native apps

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

Creates a new bind point to send version information in our existing native engine that we are exporting to rn.

I don't think it's worth the overhead to add a new module just to export this information, and semantically it makes sense for it to be a constant on the native engine since it's the version for that code.

(also changed PureComponent to be Component because it crashes rn, and makes this broken)

### CODE REVIEW

**Comment by chrisnojima** [comment]
Dont remove pure component. Either merge my redbox or cherry pick the fix

File: go/bind/keybase.go [hunk 1]
```diff
@@ -166,3 +166,8 @@ func Reset() error {
 	}
 	return nil
 }
+
+// Version returns semating version string
```
Comment (chromakode) [inline go/bind/keybase.go]: "semating"?

**Comment by chromakode** [comment]
Do we need to handle this action in a reducer? Code looks :+1:

**Comment by MarcoPolo** [comment]
This was just the first part of passing version to mobile. This may change and we might not even use this action once we decide on how we're showing this off in the app.

**Pull request merged**